src/main/groovy/edu/oregonstate/mist/webapiskeleton/SkeletonApplication.groovy

```diff
@@ -26,7 +29,11 @@ class SkeletonApplication extends Application<Configuration> {
      * @param bootstrap
      */
     @Override
-    public void initialize(Bootstrap<Configuration> bootstrap) {}
+    public void initialize(Bootstrap<Configuration> bootstrap) {
```

Comment:
Oh snap, finally a use for initialize().
I guess we'll have to make this same change to all the other apis too.
We should probably consider having a base Application class in the skeleton.